praw/models/reddit/emoji.py

```diff
@@ -127,11 +169,24 @@ def __iter__(self):
                 self._reddit, self.subreddit, emoji_name, _data=emoji_data
             )
 
-    def add(self, name, image_path):
+    def add(
+        self,
+        name,
+        image_path,
+        post_flair=True,
+        user_flair=True,
+        mod_only=False,
+    ):
         """Add an emoji to this subreddit.
 
         :param name: The name of the emoji
         :param image_path: A path to a jpeg or png image.
+        :param post_flair: (boolean) Whether the emoji will appear
+            in post flair. (Default: ``True``)
+        :param user_flair: (boolean) Whether the emoji will appear
+            in user flair. (Default: ``True``)
```

Comment:
This wording is confusing to me, but I'm not very familiar with the emoji stuff so that may be why. Specifically I don't understand what it means for an emoji to "appear" in user/post flair. Does that mean that it (will/won't) appear as an option in some list of choices? My understanding was that it controls whether the emoji is allowed to be used to flair a user or to flair a post.

I ask this because it may just be my personal experience that makes me understand, or maybe the documentation needs clarifying. In the latter case, it would have to be updated both here and in `update()`.

Reply:
I think I have something a bit better, but I'm open to suggestions. I was just using what Reddit as using, but it was pretty bad too.